lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -150,13 +157,28 @@ static bool validateCodingKeysEnum(DerivedConformance &derived,
   for (auto *varDecl : derived.Nominal->getStoredProperties()) {
     if (!varDecl->isUserAccessible())
       continue;
-
-    properties[getVarNameForCoding(varDecl)] = varDecl;
+    // When we have invalid redeclaration we may endup with duplicated
+    // members here.
```

Comment:
How about we say `redeclarations are not relevant to coding keys validation` or simply `skip redeclarations`?